src/modules/Support/html_admin/mod_support_tickets.html.twig

```diff
@@ -133,7 +133,7 @@
                             <span class="bg-warning text-white avatar">{{ statuses.on_hold }}</span>
                         </div>
                         <div class="col">
-                            <div class="font-weight-medium">{{ 'Tickets waiting for client reply'|trans }}</div>
+                            <div class="font-weight-medium">{{ 'Staff reply'|trans }}</div>
```

Comment:
I think this change will increase confusion as it no longer is obvious what the counter is showing.
For someone who's not familiar with the system it could mean tickets waiting for replies, a total number of replies that have been received, or something else entirely.

![image](https://github.com/user-attachments/assets/cfa2b287-464a-410c-941a-6a20f7ba0938)

Reply:
What about "Waiting for staff" and "Waiting for client"?

Reply:
I've revised it slightly. Please let me know if it is clearer now.

Reply:
That works for me, thanks!